flow/packet-headers/rsvp.yaml

```diff
@@ -0,0 +1,1222 @@
+              It MUST be set to zero on transmission and MUST be ignored on receipt.              
+            format: integer
+            default: 0
+            length: 16
+            features: [count]
+          x-field-uid: 1
+        l3pid:
+          x-field-pattern:
+            description: >-
+              An identifier of the layer 3 protocol using this path. 
+              Standard Ethertype values are used.             
+            format: integer
+            default: 0
+            length: 16
+            features: [count]
+          x-field-uid: 2
+    Flow.RSVP.PathObjectsClassSessionAttribute:
+      description: >-
+          C-Type is specific to a class num.
+      type: object 
+      properties:
+        c_type: 
+          $ref: '#/components/schemas/Flow.RSVP.PathObjectsSessionAttributeCType'
+          x-field-uid: 1 
+    Flow.RSVP.PathObjectsSessionAttributeCType:
+      description: >-
+        Object for SESSION_ATTRIBUTE class.
+        Currently supported c-type is LSP_Tunnel (1) and LSP_Tunnel (7).
+      type: object
+      properties:
+        choice:
+          type: string
+          x-field-uid: 1
+          default: lsp_tunnel
+          x-enum:
+            lsp_tunnel:
+              x-field-uid: 1
+            lsp_tunnel_ra:
+              x-field-uid: 2
+        lsp_tunnel:
+          $ref: '#/components/schemas/Flow.RSVP.PathSessionAttributeLspTunnel'
+          x-field-uid: 2
+        lsp_tunnel_ra:
+          $ref: '#/components/schemas/Flow.RSVP.PathSessionAttributeLspTunnelRa'
+          x-field-uid: 3
+    Flow.RSVP.PathSessionAttributeLspTunnel:
+      description: >-
+        SESSION_ATTRIBUTE class = 207, LSP_TUNNEL_RA C-Type = 7, resource affinity information.
+      type: object
+      properties:
+        setup_priority:
+          description: >-
+            The priority of the session with respect to taking resources,in the range of 0 to 7. The value 0 is the highest priority.
+            The Setup Priority is used in deciding whether this session can preempt another session.
+          type: integer
+          format: uint32
+          default: 7
+          maximum: 7
+          x-field-uid: 1
+        holding_priority:
+          description: >-
+            The priority of the session with respect to holding resources,in the range of 0 to 7. The value 0 is the highest priority.
+            The Setup Priority is used in deciding whether this session can preempt another session.
+          type: integer
+          format: uint32
+          default: 7
+          maximum: 7
+          x-field-uid: 2
+        flags:
+          description: >-
+              0x01  Local protection desired, 0x02  Label recording desired, 0x04  SE Style desired
+          $ref: '#/components/schemas/Flow.RSVPLspTunnel.Flag'
+          x-field-uid: 3
+        name_length:
+          description: >-
+            The length of the display string before padding, in bytes.          
+          type: string
```

Comment:
Not sure why this is not uint32 with proper length ,if specified.

Reply:
Yes. It should be. Fixed.